src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -237,22 +237,118 @@ export const TradeConfirm = () => {
   const isFeeRatioOverThreshold =
     networkFeeToTradeRatioPercentage > networkFeeToTradeRatioPercentageThreshold
 
+  const header: JSX.Element = useMemo(
+    () => (
+      <>
+        <Card.Header px={0} pt={0}>
+          <WithBackButton handleBack={handleBack}>
+            <Card.Heading textAlign='center'>
+              <Text
+                translation={
+                  status === TxStatus.Confirmed ? 'trade.complete' : 'trade.confirmDetails'
+                }
+              />
+            </Card.Heading>
+          </WithBackButton>
+        </Card.Header>
+        <Divider />
+      </>
+    ),
+    [handleBack, status],
+  )
+
+  const tradeWarning: JSX.Element | null = useMemo(() => {
+    const tradeWarningElement = (
+      <Flex direction='column' gap={2}>
+        {fees?.tradeFeeSource === 'Thorchain' && (
+          <Alert status='info' width='auto' fontSize='sm'>
+            <AlertIcon />
+            <Stack spacing={0}>
+              <AlertTitle>{translate('trade.slowSwapTitle', { protocol: 'THORChain' })}</AlertTitle>
+              <AlertDescription lineHeight='short'>
+                {translate('trade.slowSwapBody')}
+              </AlertDescription>
+            </Stack>
+          </Alert>
+        )}
+        {trade.buyAsset.assetId === thorchainAssetId && (
+          <Alert status='info' width='auto' mb={3} fontSize='sm'>
+            <AlertIcon />
+            <Stack spacing={0}>
+              <AlertDescription lineHeight='short'>
+                {translate('trade.intoRUNEBody')}
+              </AlertDescription>
+            </Stack>
+          </Alert>
+        )}
+      </Flex>
+    )
+    const shouldRenderWarnings = tradeWarningElement.props.children.type !== null
```

Comment:
`tradeWarningElement.props.children.type` will always evaluate to undefined, we should check for the truthiness of at least one of the items of the `children` array

Reply:
Right you are, good catch!

Fixed in https://github.com/shapeshift/web/pull/3376/commits/341283cab62f9848ccc4af1455ae0148311413ae